src/components/buttons/ScrollableButton/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+
+import { Colors, FontStyles } from '@/constants/theme';
+import { fontMaker } from '@/utils/fonts';
+
+export default StyleSheet.create({
+  button: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    backgroundColor: Colors.ActionBlue,
+    borderRadius: 100,
+    paddingHorizontal: 12,
+    paddingVertical: 10,
+    shadowColor: Colors.Black.Pure,
+    shadowOffset: { width: 2, height: 2 },
+    shadowOpacity: 0.1,
+    shadowRadius: 12,
+    elevation: 4,
+  },
+  smallButton: {
+    paddingHorizontal: 10,
+  },
+  disabled: {
+    opacity: 0.2,
+  },
+  text: {
+    ...FontStyles.Body2,
+    ...fontMaker({ color: Colors.White.Pure }),
```

Comment:
this wont work, check that fontMaker has default values that override body2 props. I think you can do 
```suggestion
    ...FontStyles.Body2,
    color: Colors.White.Pure,
```
because the problematic ones with fonts are fontWeight, fontSize and fontStyle. Or you can put `body2` to the text type, and only color here